Meilensteine/Meilenstein_04/frontend/test/BubbleSortLogic.test.js

```diff
@@ -1,88 +1,48 @@
-import fs from 'fs';
-import path from 'path';
+// import fs from 'fs';
+// import path from 'path';
 import {
-  expect, describe, test, beforeEach,
+  expect, describe, test, beforeEach, jest,
 } from '@jest/globals';
 import BubbleSort from '../src/BubbleSortLogic';
 
-const bubblesortHtml = fs.readFileSync(path.resolve(__dirname, '../src/BubbleSort.html'), 'utf8');
-
+// const bubblesortHtml = fs.readFileSync
+// (path.resolve(__dirname, '../src/BubbleSort.html'), 'utf8');
 describe('BubbleSort', () => {
   describe('helper functions', () => {
-    // "Dumme" Nutzereingaben
-    let bubble;
-    let mockStateChangeCallback;
+    let mockStateChangeCallback = jest.fn();
+    let bubble = new BubbleSort(jest.fn(), mockStateChangeCallback);
```

Comment:
```suggestion
    let bubble = new BubbleSort({}, mockStateChangeCallback);
```
Ein EventReporter ist keine Funktion sondern ein Objekt